src/cmd/ksh93/tests/b_print.sh

```diff
@@ -104,3 +104,24 @@ then
 else
     log_error 'print -s -f fails'
 fi
+
+# ======
+# Check that I/O errors are detected (bug #1093)
```

Comment:
We have several channels to track bugs (Red hat bugzilla, GitHub etc.). Also, bug tracking mechanisms may change over time. While mentioning bug id, it's good to have reference to actual platform (in this case GitHub).

Reply:
Done.